internal/controller/user_handler.go

```diff
@@ -0,0 +1,76 @@
+package todo
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	models "github.com/daniyalumer/todo-list-go-chi/internal/models"
+	service "github.com/daniyalumer/todo-list-go-chi/internal/service"
+	"github.com/go-chi/chi/v5"
+)
+
+func UserRoutes() chi.Router {
+	r := chi.NewRouter()
+
+	r.Get("/get", GetUserHandler)
+	r.Post("/post", CreateUserHandler)
+	r.Delete("/delete", DeleteUserHandler)
+
+	return r
+}
+
+func GetUserHandler(w http.ResponseWriter, r *http.Request) {
+	message, users := service.ReadUsers()
+	w.Header().Set("Content-Type", "application/json")
+
+	response := struct {
+		Message string        `json:"message"`
+		Users   []models.User `json:"todos"`
+	}{
+		Message: message,
+		Users:   users,
+	}
+
+	json.NewEncoder(w).Encode(response)
```

Comment:
Create a separate function that can receive an object and set in the response writer. It will remove the redundant code like setting headers and parsing each time